src/renderer/store/modules/delegate.js

```diff
@@ -25,6 +25,20 @@ export default {
       }
 
       return state.delegates[network.id][address]
+    },
+
+    byPublicKey: (state, _, __, rootGetters) => publicKey => {
+      const network = rootGetters['session/network']
+
+      if (!network || !state.delegates[network.id]) {
```

Comment:
Could probably also check to make sure publicKey is set. Unlikely to happen but saves the loop in case it ever is a scenario in the future elsewhere